livepeer_cli runtime error

**Describe the bug**

`livepeer_cli` having issues on dev. Manually setting `w.offchain = true` in `func (w *wizard) checkNet() {}` fixes the issue temporarily.

**To Reproduce**

1. Run `./livepeer -datadir $HOME/lp-data-1 -broadcaster -transcodingOptions P144p30fps16x9,P576p30fps16x9 -orchAddr localhost:1998 -authWebhookUrl http://localhost:3004/api/stream/hook`
2. Run `./livepeer -datadir $HOME/lp-data-2 -orchestrator -transcoder -cliAddr localhost:1999 -serviceAddr localhost:1998`
3. Run `./livepeer_cli -http 7935`
4. See error:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x47fa97e]

goroutine 1 [running]:
math/big.(*Int).Int64(...)
	/usr/local/go/src/math/big/int.go:372
main.(*wizard).delegatorStats(0xc0000f1ec0)
	/Users/angie/git/go-livepeer/cmd/livepeer_cli/wizard_stats.go:228 +0x8e
main.(*wizard).stats(0xc0000f1ec0, 0xc0000d0000)
	/Users/angie/git/go-livepeer/cmd/livepeer_cli/wizard_stats.go:78 +0xb3c
main.(*wizard).run(0xc0000f1ec0)
	/Users/angie/git/go-livepeer/cmd/livepeer_cli/livepeer_cli.go:146 +0x3ce
main.main.func1(0xc000328000, 0xc000328000, 0xc000557ac7)
	/Users/angie/git/go-livepeer/cmd/livepeer_cli/livepeer_cli.go:56 +0x551
github.com/urfave/cli.HandleAction(0x50ba080, 0x528dcf0, 0xc000328000, 0xc000312ba0, 0x0)
	/Users/angie/go/pkg/mod/github.com/urfave/cli@v1.20.0/app.go:490 +0xc8
github.com/urfave/cli.(*App).Run(0xc0002bc820, 0xc0000ba180, 0x3, 0x3, 0x0, 0x0)
	/Users/angie/go/pkg/mod/github.com/urfave/cli@v1.20.0/app.go:264 +0x58c
main.main()
	/Users/angie/git/go-livepeer/cmd/livepeer_cli/livepeer_cli.go:62 +0x32b
```

**Expected behavior**
An interactive livepeer_cli 

**Desktop (please complete the following information):**
Terminal

**Additional Notes**
(From @j0sh ) Looking at delegatorStats (the function where it's crashing) there is indeed an offchain check, so whatever w.offchain is, it's not being set correctly.  Yep - looks like the EthChainID just writes a "0" for offchain when it should be "offchain". Either that or the checkNet function needs to be updated to check for  nID == "0"

--- a/cmd/livepeer_cli/livepeer_cli.go
+++ b/cmd/livepeer_cli/livepeer_cli.go
@@ -173,5 +173,5 @@ var DevenvChainID = "54321"
 func (w *wizard) checkNet() {
 	nID := httpGet(fmt.Sprintf("http://%v:%v/EthChainID", w.host, w.httpPort))
 	w.testnet = nID == RinkebyChainID || nID == DevenvChainID
-	w.offchain = nID == "offchain"
+	w.offchain = nID == "0"
 }

--- a/server/cliserver_test.go
+++ b/server/cliserver_test.go
@@ -52,17 +52,39 @@ func TestGetStatus(t *testing.T) {
 }
 
 func TestGetEthChainID(t *testing.T) {
-	srv := newMockServer()
-	defer srv.Close()
-	res, err := http.Get(fmt.Sprintf("%s/EthChainID", srv.URL))
+	require := require.New(t)
 	assert := assert.New(t)
-	req := require.New(t)
-	req.Nil(err)
+
+	// test offchain
+	srv := newMockServer()
+	res, err := http.Get(fmt.Sprintf("%s/EthChainID", srv.URL))
+	require.Nil(err)
 	assert.Equal(http.StatusOK, res.StatusCode)
-	defer res.Body.Close()
 	body, err := ioutil.ReadAll(res.Body)
-	req.Nil(err)
+	require.Nil(err)
 	assert.Equal("0", string(body))
+	defer res.Body.Close()
+	defer srv.Close()
+
+	// test onchain
+	dbh, dbraw, err := common.TempDB(t)
+	defer dbh.Close()
+	defer dbraw.Close()
+	require.Nil(err)
+	err = dbh.SetChainID(big.NewInt(1))
+	require.Nil(err)
+	n, _ := core.NewLivepeerNode(&eth.StubClient{}, "./tmp", dbh)
+	s := NewLivepeerServer("127.0.0.1:1938", n, true)
+	mux := s.cliWebServerHandlers("addr")
+	srv = httptest.NewServer(mux)
+	defer srv.Close()
+	res, err = http.Get(fmt.Sprintf("%s/EthChainID", srv.URL))
+	require.Nil(err)
+	assert.Equal(http.StatusOK, res.StatusCode)
+	body, err = ioutil.ReadAll(res.Body)
+	defer res.Body.Close()
+	require.Nil(err)
+	assert.Equal("1", string(body))
 }
 
 func TestGetContractAddresses(t *testing.T) {

--- a/server/webserver.go
+++ b/server/webserver.go
@@ -1077,15 +1077,11 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 			w.Write([]byte("0"))
 			return
 		}
-		be, err := s.LivepeerNode.Eth.Backend()
+		chainID, err := s.LivepeerNode.Database.ChainID()
 		if err != nil {
-			glog.Errorf("Error getting eth backend: %v", err)
+			glog.Errorf("Error getting eth network ID err=%v", err)
 			w.WriteHeader(http.StatusInternalServerError)
 			return
-		}
-		chainID, err := be.ChainID(r.Context())
-		if err != nil {
-			glog.Errorf("Error getting eth network ID: %v", err)
 		}
 		w.Write([]byte(chainID.String()))
 	})

